<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetClass</class>
 <widget class="QWidget" name="WidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>315</width>
    <height>388</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Choose OS</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QCheckBox" name="windowsCheckbox">
          <property name="text">
           <string>Terminal</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="linuxCheckbox">
          <property name="text">
           <string>Javascript</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="macCheckbox">
          <property name="text">
           <string>Mac</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Choose GPU manufacturers</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QCheckBox" name="NvidiaCheckBox">
          <property name="text">
           <string>NVIDIA</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="IntelCheckbox">
          <property name="text">
           <string>Intel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="AMDCheckbox">
          <property name="text">
           <string>AMD</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="IBMCheckbox">
          <property name="text">
           <string>IBM</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Select valid programming language</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QRadioButton" name="QtRadio">
        <property name="text">
         <string>Qt</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="CppRadio">
        <property name="text">
         <string>C++</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="htmlRadio">
        <property name="text">
         <string>HTML</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="okButton">
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="revealButton">
     <property name="text">
      <string>Correct Answers</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Widget.qrc"/>
 </resources>
 <connections/>
</ui>
